application/testing/CMakeLists.txt

```diff
@@ -177,12 +177,13 @@ f3d_test(NAME TestGroupGeometriesColoring DATA mb/recursive ARGS --group-geometr
 
 # Test missing symbol plugin using the libf3d
 if(WIN32)
-      f3d_test(NAME TestPluginInvalid ARGS --verbose --load-plugins "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/f3d.dll" NO_BASELINE REGEXP "Cannot find init_plugin symbol in library")
+  set(_libname "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/f3d.dll")
```

Comment:
What about

```cmake
set(_libname ${CMAKE_SHARED_MODULE_PREFIX}f3d${CMAKE_SHARED_MODULE_SUFFIX})
```

Reply:
what is the point ? this is not going to change and I cant factorise because of the `lib`

Reply:
Please note the `${CMAKE_SHARED_MODULE_PREFIX}` being `lib` on Unix and empty on windows.

Reply:
!!